packages/client/hmi-client/src/components/model-template/tera-model-template.vue

```diff
@@ -1,150 +1,163 @@
 <template>
-	<tera-infinite-canvas
-		@save-transform="saveTransform"
-		@mouseenter="setMouseOverCanvas(true)"
-		@mouseleave="setMouseOverCanvas(false)"
-		@drop="onDrop"
-		@dragover.prevent
-		@dragenter.prevent
-		@focus="() => {}"
-		@blur="() => {}"
-	>
-		<template #foreground>
-			<aside>
-				<section v-if="model?.header?.schema_name">
-					<header>Model framework</header>
-					<h5>{{ model.header.schema_name }}<i class="pi pi-info-circle"></i></h5>
-				</section>
-				<section>
-					<header>Model templates</header>
-					<ul>
-						<li v-for="(_, index) in 5" :key="index">
-							<tera-model-template-card :card="newCard" draggable="true" />
-						</li>
-					</ul>
-				</section>
-				<section class="trash">
-					<i class="pi pi-trash"></i>
-					<div>Drag items here to delete</div>
-				</section>
-			</aside>
-		</template>
-		<template #data>
-			<tera-canvas-item
-				v-for="(card, index) in cards"
+	<section class="card-container">
+		<section class="card" ref="cardRef">
+			<div class="draggable"><i class="pi pi-pause" /></div>
+			<main>
+				<header>
+					<span @click="isEditingName = true" v-if="!isEditingName">
+						{{ card.name }}
+					</span>
+					<InputText v-else size="small" type="text" v-model="name" @keyup.enter="updateName" />
+				</header>
+				<section>Diagram/Equations</section>
+			</main>
+			<Button icon="pi pi-ellipsis-v" rounded text />
+		</section>
+		<ul>
+			<li
+				v-for="(variable, index) in fakeVariables"
+				class="port"
 				:key="index"
```

Comment:
Avoid `:key=index` if you can, it can have unexpected reactivity issues because object in the list can change but the key is the same.